tools/inference/torch_inference.py

```diff
@@ -0,0 +1,92 @@
+"""Anomalib Torch Inferencer Script.
+
+This script performs torch inference by reading model config files and weights
+from command line, and show the visualization results.
+"""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+
+from anomalib.data.utils import (
+    generate_output_image_filename,
+    get_image_filenames,
+    read_image,
+)
+from anomalib.deploy import TorchInferencer
+from anomalib.post_processing import Visualizer
+
+
+def get_args() -> Namespace:
+    """Get command line arguments.
+
+    Returns:
+        Namespace: List of arguments.
+    """
+    parser = ArgumentParser()
+    parser.add_argument("--config", type=Path, required=True, help="Path to a config file")
+    parser.add_argument("--weights", type=Path, required=True, help="Path to model weights")
+    parser.add_argument("--meta_data", type=Path, required=False, help="Path to a JSON file containing the metadata.")
```

Comment:
What if we want to deploy the model from the .pth file instead of the .ckpt file? In that case we would need the meta_data dictionary. This is related to another concern that I'm having. The torch inferencer now still uses lightning classes under the hood. We could consider making the torch inferencer pytorch-only, and have it load a .pth model instead of .ckpt model. This way we would not need to import pytorch lightning to run torch inference and keep the different inferencers better separated. The advantage for users would be that it's more in line with how they will likely incorporate their trained models into their software.